cucumber-jparest/src/main/java/uk/gov/homeoffice/digital/sas/cucumberjparest/scenarios/stepdefinitions/ApiSteps.java

```diff
@@ -96,7 +95,7 @@ public void personaRetrievesResourcesFromTheService(Persona persona, String reso
    */
   @When("{persona} GETs {string}{service}")
   public void personaGetsUrlFromService(Persona persona, String path, String service) {
-    JpaRestApiResponse apiResponse = this.jpaRestApiClient.get(persona, service, path);
+    JpaRestApiResponse apiResponse = this.jpaRestApiClient.get(persona, null, service, path);
```

Comment:
I'm not sure why we added this param to always pass null and then grab it from the system property.

Reply:
tenantId method argument removed as discussed yesterday